src/main/kotlin/no/nav/tilleggsstonader/sak/vedtak/domain/ÅrsakAvslag.kt

```diff
@@ -1,13 +1,17 @@
 package no.nav.tilleggsstonader.sak.vedtak.domain
 
-enum class ÅrsakAvslag {
-    INGEN_AKTIVITET,
-    IKKE_I_MÅLGRUPPE,
-    INGEN_OVERLAPP_AKTIVITET_MÅLGRUPPE,
-    MANGELFULL_DOKUMENTASJON,
-    HAR_IKKE_UTGIFTER,
-    RETT_TIL_UTSTYRSSTIPEND,
-    HAR_IKKE_MERUTGIFTER,
-    RETT_TIL_BOSTØTTE,
-    ANNET,
```

Comment:
Ser at vi flere steder har behov for en ´displayName` på denne måten. 

I `BehandlingStatus` og `StegType` er det løst ved å lage en funksjon som utleder strengen:

```
    fun visningsnavn(): String =
        this.name
            .replace('_', ' ')
            .lowercase()
            .replaceFirstChar { it.uppercase() }
```